Sources/Node/Convertible/Date+Convertible.swift

```diff
@@ -0,0 +1,41 @@
+import Foundation
+
+extension Date: NodeConvertible {
+    /**
+     Turn the Date into a node
+     
+     - throws: if convertible can not create a Node
+     - returns: a node if possible
+     */
+    public func makeNode(context: Context) throws -> Node {
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "yyyy-MM-dd HH:mm:ss"
```

Comment:
This might be the inner iOS developer in me :), but I generally prefer having a static date formatter initialised once, if the format stays the same, as it is way more performant. The same formatter can be used for the init method underneath.